mdn-drafts/Screens.md

```diff
@@ -0,0 +1,89 @@
+---
+recipe: api-interface
+title: 'Screens'
+mdn_url: /en-US/docs/Web/API/Screens
+specifications: https://webscreens.github.io/window-placement
+browser_compatibility: api.Screens
+---
+
+## Description
+
+
+The `Screens` interface of the Window Placement API provides multi-screen information and change events.
+
+## Constructor
+
+None.
+
+## Properties
+
+**`Screens.screens`**
+
+Returns an array of `ScreenAdvanced` objects, which describe each connected screen.
+
+**`Screens.currentScreen`**
+
+Returns a `ScreenAdvanced` object for the current screen.
+
+## Eventhandlers
+
+**`Screens.onchange`**
```

Comment:
This event has been split, see the explainer as needed; Here's some updated content you may use (I wonder if we should document the new `Screen.onchange` itself somewhere here):

**`Screens.onscreenschange`**

Called when `screens` changes.

Note that this is not called on changes to the attributes of individual screens. `Screen.onchange` can be used to observe those changes.

**`Screens.oncurrentscreenchange`**

Called when any attribute on `currentScreen` changes.

Reply:
Fixed.

>I wonder if we should document the new Screen.onchange itself somewhere here

Yes. Infact, both `Screen.onchange` and `Screen.isExtended` both need to be documented, but not in PR.

Reply:
We're halfway there, Screen.isExtended is documented in the new isExtended.md